doc/about.rst

```diff
@@ -0,0 +1,42 @@
+=====
```

Comment:
Are you confident about the benefits of adding this file ? Each new file requires maintenance

Reply:
I have no strong opinion.

I will integrate this part into the home page.